lib/elixir_conf_africa_web/live/home_live/index.ex

```diff
@@ -1,14 +1,55 @@
 defmodule ElixirConfAfricaWeb.HomeLive.Index do
   use ElixirConfAfricaWeb, :live_view
   alias ElixirConfAfrica.Events
+  alias ElixirConfAfrica.TicketTypes
 
   def mount(_params, _session, socket) do
     elixir_conf_africa_event = Events.get_elixir_conf_event_and_ticket_types()
     availabe_tickets = Events.get_all_available_tickets()
+    cart = []
+    total_price = get_total_price(cart)
 
     {:ok,
      socket
      |> assign(:available_tickets, availabe_tickets)
+     |> assign(:cart, cart)
+     |> assign(:total_price, total_price)
+     |> assign(:page_state, "details")
      |> assign(:event, elixir_conf_africa_event)}
   end
+
+  def handle_event("add_to_cart", %{"id" => id}, socket) do
+    ticket_type =
+      id
+      |> String.to_integer()
+      |> TicketTypes.get_ticket_type!()
+      |> TicketTypes.add_quantity(1)
+
+    if Enum.member?(socket.assigns.cart, ticket_type) do
+      {:noreply,
+       socket
+       |> put_flash(:error, "Ticket already in cart")}
```

Comment:
is this what should happen if the ticket is already added to cart? 🤔  In this scenario shouldn't we increment the quantity of the ticket?